### PR TITLE
duf: add Chinese translation

### PR DESCRIPTION
**Pull request by rguards**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.zh/common/duf.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+- 只显示指定的设备或挂载点：
+
+`duf {{路径/到/文件夹1 路径/到/文件夹2 ...}}`
+
+- 根据指定标准排序输出：
+
+`duf --sort {{size|used|avail|usage}}`
+
+- 显示或隐藏指定文件系统
```
Comment (einverne) [inline pages.zh/common/duf.md:22]:
```suggestion
- 显示或隐藏指定文件系统：
```

File: pages.zh/common/duf.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+`duf --{{only-fs|hide-fs}} {{tmpfs|vfat|ext4|xfs}}`
+
+- 根据键排序输出：
+
+`duf --sort {{mountpoint|size|used|avail|usage|inodes|inodes_used|inodes_avail|inodes_usage|type|filesystem}}`
+
+- 更改主题（如果 `duf` 未能使用正确的主题）
```
Comment (einverne) [inline pages.zh/common/duf.md:30]:
```suggestion
- 更改主题（如果 `duf` 未能使用正确的主题）：
```

File: pages.zh/common/duf.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- 列出所有（如伪文件系统，重复文件系统或不可访问的文件系统）：
+
+`duf --all`
+
+- 只显示指定的设备或挂载点：
+
+`duf {{路径/到/文件夹1 路径/到/文件夹2 ...}}`
+
+- 根据指定标准排序输出：
```
Comment (einverne) [inline pages.zh/common/duf.md:18]:
```suggestion
- 根据指定条件排序输出：
```

**Review by einverne — CHANGES_REQUESTED** [review]
Hi @rguards thank you for the translation, I left some suggestions.

File: pages.zh/common/duf.md (hunk 2 above)
Comment (tricantivu) [reply pages.zh/common/duf.md:30]: Is the space before the colon necessary?

Comment (kbdharun) [reply pages.zh/common/duf.md:30]: > Is the space before the colon necessary?

Yeah, it doesn't seem to be the case in other similar `zh` pages.

For example:

https://github.com/tldr-pages/tldr/blob/d0d27e1f6e905e006a62826cfab8b85140e0ea9c/pages.zh/common/more.md?plain=1#L22

**Review by einverne — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Comment by kbdharun** [comment]
**Note:** I closed/reopened the pull request to retrigger the stuck CLA check; to merge this PR.

**Pull request merged**